doc/interoperability/cycloneds.md

```diff
@@ -0,0 +1,77 @@
+Properties with the same key may appear more than once.
+Names and values are case-sensitive.
+
+| Property         | Description |
+|------------------|-------------|
+| `nix:store_path` | A Nix store path for the given component. This property should be contextualized by additional properties that describe the production of the store path, such as those from the `nix:narinfo:` and `nix:fod` namespaces. |
+
+
+| Namespace     | Description |
+|---------------|-------------|
+| [`nix:narinfo`](#sec-interop.cylonedx-narinfo) | Namespace for properties that are specific to how a component is archived in a binary cache. |
+| [`nix:fod`](#sec-interop.cylonedx-fod) | Namespace for properties that describe a Fixed-Output-Derivation. |
+
+
+### `nix:narinfo` {#sec-interop.cylonedx-narinfo}
+
+Narinfo properties describe component archives that may be available from binary caches.
+The `nix:narinfo` properties should be accompanied by a `nix:store_path` property within the same property list.
+
+| Property                  | Description |
+|---------------------------|-------------|
+| `nix:narinfo:store_path`  | Store path for the given store component. |
+| `nix:narinfo:url`         | URL path component. |
+| `nix:narinfo:nar_hash`    | Hash of the file system object part of the component when serialized as a Nix Archive. |
+| `nix:narinfo:nar_size`    | Size of the component when serialized as a Nix Archive. |
+| `nix:narinfo:compression` | The compression format that component the archive is in. |
+| `nix:narinfo:file_hash`   | A digest for the compressed component archive itself, as opposed to the data contained within. |
+| `nix:narinfo:file_size`   | The size of the compressed component archive itself. |
+| `nix:narinfo:deriver`     | The path to the derivation from which this component is produced. |
+| `nix:narinfo:system`      | The hardware and software platform on which this component is produced. |
+| `nix:narinfo:sig`         | Signatures claiming that this component is what it claims to be. |
+| `nix:narinfo:ca`          | Content address of this store object's file system object, used to compute its store path. |
+| `nix:narinfo:references`  | A whitespace separated array of store paths that this component references. |
+
+### `nix:fod` {#sec-interop.cylonedx-fod}
+
+FOD properties describe a Fixed-Output-Derivation as a method and attributes passed to a function that implements the given method.
+The `nix:fod:method` property is required and must be accompanied by a `nix:store_path` property within the same property list.
+All other properties in this namespace are method-specific.
+
+| Property         | Description |
+|------------------|-------------|
+| `nix:fod:method` | Nixpkg function that produces this FOD. Required. Examples: `"fetchzip"`, `"fetchgit"` |
+| `nix:fod:name`   | Derivation name, present when method is `"fetchzip"` |
+| `nix:fod:ref`    | Git ref, present when method is `"fetchgit"` |
```

Comment:
Please link to the Git man pages (not the manual!) on ref and rev, this is known to throw people off.